vignettes/tech-dm-zoom.Rmd

```diff
@@ -60,9 +60,8 @@ flights_zoomed
 
 flights_zoomed_mutate <-
   flights_zoomed %>%
-  mutate(am_pm_dep = if_else(dep_time < 1200, "am", "pm")) %>%
-  # in order to see our changes in the output we use `select()` for reordering the columns
-  select(year:dep_time, am_pm_dep, everything())
+  mutate(am_pm_dep = if_else(dep_time < 1200, "am", "pm"), .after = "dep_time")
```

Comment:
This isn't going to work yet with zooming, #1022.

Reply:
Ah, I see. I've reverted this change.